rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/factory/AbstractExamplesFactory.java

```diff
@@ -135,43 +135,43 @@ public Operation generateExample(final Class<? extends Operation> opClass) throw
 
     private Object getExampleValue(final Class clazz, final int uniqueId) {
         Object value;
-        if (null == clazz) {
-            value = null;
-        } else if (Character.class.equals(clazz) || char.class.equals(clazz)) {
-            value = (char) uniqueId;
-        } else if (String.class.equals(clazz) || Object.class.equals(clazz)) {
-            value = String.valueOf(uniqueId);
-        } else if (Integer.class.equals(clazz) || int.class.equals(clazz)) {
-            value = uniqueId;
-        } else if (Double.class.equals(clazz) || double.class.equals(clazz)) {
-            value = (double) uniqueId + 0.1;
-        } else if (Long.class.equals(clazz) || long.class.equals(clazz)) {
-            value = (long) uniqueId;
-        } else if (Float.class.equals(clazz) || float.class.equals(clazz)) {
-            value = (float) uniqueId;
-        } else if (Date.class.equals(clazz)) {
-            value = new Date(System.currentTimeMillis() - 10000 + uniqueId);
-        } else if (boolean.class.equals(clazz)) {
-            value = uniqueId % 2 == 0;
-        } else if (BinaryOperator.class.equals(clazz)) {
-            value = new First();
-        } else {
-            try {
-                if (clazz.isEnum()) {
-                    final List l = Arrays.asList(clazz.getEnumConstants());
-                    if (!l.isEmpty()) {
-                        value = Enum.valueOf(clazz, l.get(0).toString());
-                    } else {
-                        value = clazz.newInstance();
-                    }
+        try {
+            if (null == clazz) {
+                value = null;
+            } else if (Character.class.equals(clazz) || char.class.equals(clazz)) {
+                value = (char) uniqueId;
+            } else if (String.class.equals(clazz) || Object.class.equals(clazz)) {
+                value = String.valueOf(uniqueId);
+            } else if (Integer.class.equals(clazz) || int.class.equals(clazz)) {
+                value = uniqueId;
+            } else if (Double.class.equals(clazz) || double.class.equals(clazz)) {
+                value = (double) uniqueId;
+            } else if (Long.class.equals(clazz) || long.class.equals(clazz)) {
+                value = (long) uniqueId;
+            } else if (Float.class.equals(clazz) || float.class.equals(clazz)) {
+                value = (float) uniqueId;
+            } else if (Date.class.equals(clazz)) {
+                value = new Date(System.currentTimeMillis() - 10000 + uniqueId);
+            } else if (boolean.class.equals(clazz)) {
+                value = uniqueId % 2 == 0;
+            } else if (BinaryOperator.class.equals(clazz)) {
+                value = new First();
+            } else if (clazz.isEnum()) {
+                final List l = Arrays.asList(clazz.getEnumConstants());
+                if (!l.isEmpty()) {
+                    value = Enum.valueOf(clazz, l.get(0).toString());
```

Comment:
While not really part of the changes in this PR, this stuff wasn't covered before and I'm unsure of what it's doing and if it will work properly.

Reply:
True this section from what I can tell is mostly to do with creating an Entity with the `vertex` type defined in the schema e.g. if it was meant to be an `Integer` this generates an Entity with that value for its `vertex` field. With that I can only assume this bit is for if the schema has defined an `Enum` as the type for `vertex` it creates one with the first value in the `Enum`.

Reply:
That makes sense. I don't much like this code as it also uses rawtypes when `Class<?>` should be used. I think I might raise a new issue to look at those aspects of this class. Not something which needs to be done in this PR.